.github/workflows/pipeline.yml

```diff
@@ -1,10 +1,7 @@
 on:
   push:
-    branches:
```

Comment:
Later we might introduced more specialised actions for different branch types, for now I just want testing to keep happening on both main and devel as well as on feature branches, release branches etc.